src/exposure.jl

```diff
@@ -0,0 +1,177 @@
+const DEGRADATION_RATE = Param(0.5; bounds=(0.0, 100.0))
+const PULSE_LEVEL = Param(20.0; bounds=(0.0, 100.0))
+const POPULATION_THRESHOLD = Param(20.0; bounds=(0.0, 100.0))
+const CROP_OCCUPANCY = Param(5.0; bounds=(0.0, 10.0))
+const ROTATION_NUMBER = Param(1.0; bounds=(0.0, 1000.0))
+const ROTATION_SIZE = Param(1.0; bounds=(0.0, 1000.0))
+
+"""
+Abstract supertype of `CellRule` for rules of exposure dynamics
+"""
+
+abstract type Exposure{R,W} <: CellRule{R,W} end
```

Comment:
The types you have suggested might not work because the threshold concept and rotating concept are not mutually exclusive. But I would suggest rotation by default because no rotation is just a special case of rotation where the series is just `(treatment1, treatment1, treatment1,...)`

Reply:
Ok. FYI these are just renames of the types already in this PR.

I'm also wondering if these actually just `ExponentialGrowth` with this rotation/threshold/rotation+threshol treatment regime as separate, very simple rules? Should the degradation be a separate rule from the means of applying the pesticide?

Reply:
Oh, very good point.
The assumption is that treatment always degradates. So we should consider the degradation rule as separate (which is the same as the exponential growth with a negative rate).

Then we would have the pulse rule which would be very simple based on theshold, And a rotation+pulsethreshold rule...